sample-code/examples/ruby/cucumber_ios/config/cucumber.yml

```diff
@@ -0,0 +1,6 @@
+# config/cucumber.yml
+##YAML Template
+---
+ipadsim: IDEVICE=ipad IDEVICENAME='ipad simulator'
```

Comment:
I intended to use it in step definitions that are different on different form factors but I didn't find an applicable case here.
I will remove it now.